### PR TITLE
Rationalise table generation

### PR DESCRIPTION
**Pull request by jrha**
* Ensure all tables that have a head have a body
* Remove stubs of old odd-even row highlighting code
* Remove redundant size checks around iterators (they will not iterate anyway)
* Use a consistent control block syntax (no endif please)
* Correct helper comments at end of long loops
* Remove commented out code blocks
* Sort out some of the indentation insanity

### CODE REVIEW

**Comment by jrha** [comment]
Probably worth reviewing this one [with white-space changes hidden](https://github.com/GOCDB/gocdb/pull/104/files?w=1).

**Comment by jrha** [comment]
How does PHP 5.5 suddenly not exist on Travis?

**Comment by jrha** [comment]
All done. Previous failure seems to have been a transient Travis issue.

**Comment by jrha** [comment]
If you want to make sure that this doesn't change anything layout wise, it's running on a VM at RAL, will email details separately.

**Comment by jrha** [comment]
:roll_eyes:  Well who can?

**Comment by jrha** [comment]
Also worth noting that this *does not* fix the problem described in #104.

**Comment by tofu-rocketry** [comment]
I don't think you mean #104, @jrha.

File: htdocs/web_portal/views/project/view_project.php [hunk 1]
```diff
@@ -50,34 +50,32 @@
...
+                <tbody>
+                    <?php
+                    foreach($params['NGIs'] as $ngi) {
+                    ?>
+                        <tr>
+                            <td>
+                                <img class="flag" src="<?php echo \GocContextPath::getPath()?>img/ngi/<?php xecho($ngi->getName()) ?>.jpg">
+                            </td>
+                            <td>
+                                <a href="index.php?Page_Type=NGI&id=<?php echo $ngi->getId() ?>"><?php xecho($ngi->getName()); ?></a>
```
Comment (tofu-rocketry) [inline htdocs/web_portal/views/project/view_project.php]: Will this conflict with the escape fixes?

Comment (jrha) [reply htdocs/web_portal/views/project/view_project.php]: Yeees? Maybe?

Comment (jrha) [reply htdocs/web_portal/views/project/view_project.php]: Turns out nope, doesn't conflict!

File: htdocs/web_portal/views/service_group/view_all.php [hunk 2]
```diff
@@ -70,38 +70,36 @@
...
+                            <a href="index.php?Page_Type=Service_Group&amp;id=<?php echo $sGroup->getId()?>">
+                            <?php xecho($sGroup->getName()); ?>
+                            </a>
+                        </td>
+                        <td>
+                            <?php xecho($sGroup->getDescription()); ?>
+                        </td>
+                        <td>
+                            <textarea readonly="true" style="height: 25px;">
+                                <?php xecho($sGroup->getScopeNamesAsString()); ?>
```
Comment (GRyall) [inline htdocs/web_portal/views/service_group/view_all.php]: php marker needs to move to front of line, or similar as:
image

**Comment by jrha** [comment]
Whoops :disappointed:, there was another instance of the same thing in `htdocs/web_portal/views/service/view_all.php`. Both are now fixed.

**Comment by GRyall** [comment]
Weirdly that one didn't show up in the same way, in my browser at least

**Pull request merged**